src/main/webapp/WEB-INF/web.xml

```diff
@@ -65,5 +65,13 @@
     <servlet-name>RegisterServlet</servlet-name>
     <url-pattern>/register</url-pattern>
   </servlet-mapping>
+  <servlet>
+  	<servlet-name>AdminServlet</servlet-name>
+  	<servlet-class>codeu.controller.AdminServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+  	<servlet-name>AdminServlet</servlet-name>
+  	<url-pattern>/admin</url-pattern>
+  </servlet-mapping>
 
```

Comment:
Good work!!It seems that you have a basic prototype working. Make sure to create a test for the Admin Servlet class. Im not sure if you're already doing this but try compiling your code in the command line to make sure it compiles. I see you modified some of the prearranged files.You might have to modify their corresponding tests classes as well to make sure your code compiles before merging into the master branch.